content/docs/next-release/opentelemetry.md

```diff
@@ -0,0 +1,114 @@
+---
+title: OpenTelemetry
+hasparent: true
+---
+
+The future Jaeger backend components will be based on [OpenTelemetry collector](https://opentelemetry.io/docs/collector/about/).
+This integration will make all OpenTelemetry Collector features available in the Jaeger backend components.
+
+{{< warning >}}
+At the moment Jaeger OpenTelemetry components are experimental.
+{{< /warning >}}
+
+The Jaeger OpenTelemetry backend components are published as Docker images:
+
+Component             | Repository
+--------------------- | ---
+**jaeger-agent**      | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/)
+**jaeger-collector**  | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/)
+**jaeger-ingester**   | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/)
+**all-in-one**        | [hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/](https://hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/)
+
+## Compatibility
+
+The Jaeger OpenTelemetry components are **almost** backward compatible legacy Jaeger components.
+
+The differences are:
+
+* Health check port changed to `13133`
+* Not all legacy Jaeger flags are exposed (e.g. health check port)
+* Exposed metrics
+
+## Configuration
+
+Jaeger OpenTelemetry components can be configured by a subset of Jaeger legacy flags (or other [configuration sources](deployment.md#configuration-3))
+and [OpenTelemetry configuration file](https://opentelemetry.io/docs/collector/configuration/).
+The OpenTelemetry configuration takes precedence over Jaeger configuration.
```

Comment:
Thought we were going to change that. Are there some technical reason for this? It's quite unintuitive that CLI flags are lower priority than anything else. The traditional order (low to high priority) is config, env, CLI.

Reply:
There is an open ticket for it https://github.com/jaegertracing/jaeger/issues/2209 once is resolved we can change the docs

Reply:
I have added a link with filtered otel issues to the warning box at the beginning of the page with a comment that configuration and behavior can change.